templates/service-role.yml

```diff
@@ -304,4 +305,4 @@ Resources:
               "Resource": "arn:aws:events:*:*:rule/*"
             }
           ]
-        }
\ No newline at end of file
+        }
```

Comment:
👍

Reply:
so it's not clear from context on the main PR page but i'm a big fan of newlines at the ends of files, and this change adds, one, i love to see it